dd-client/io_runner.sh

```diff
@@ -1,27 +1,31 @@
 #!/bin/bash
 
-#######################################################################################################################
-# Script Name   : io_runner.sh         									      		
-# Description   : Run dd profiles on the /datadir. 
-# Creation Data : 12/10/2018                                                                                          
-# Modifications : None											               		
-# Script Author : Sudarshan					      
-#######################################################################################################################
-
-TEST_SIZE="512k"
-TEST_COUNT="1000"
+blockSize=$(echo $BLOCK_SIZE)
+count=$(echo $COUNT)
```

Comment:
is an echo needed here? you can assign env to a bash var like this too - count=${COUNT}? moreover, retry count, duration can be provided defaults in here too as count=${COUNT:=1024}-- though I'd leave it to you/optional.